internal/byok/command_create.go

```diff
@@ -138,12 +197,18 @@ func isAWSKey(key string) bool {
 	return keyArn.Service == "kms" && strings.HasPrefix(keyArn.Resource, "key/")
 }
 
+func isGCPKey(key string) bool {
```

Comment:
```suggestion
func isGcpKey(key string) bool {
```

Reply:
Is there a particular reason we don't want to stick with golang naming [conventions](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) here?

Reply:
Cool, didn't know this was a thing! Carry on...

Reply:
Looks like we agreed to lowercase this after all in a later comment
```suggestion
func isGcpKey(key string) bool {
```